src/TLuaInterpreter.cpp

```diff
@@ -3304,7 +3304,7 @@ int TLuaInterpreter::setTextFormat(lua_State* L)
         }
     }
 
-    if (windowName.isEmpty() || windowName.compare(QStringLiteral("main"), Qt::CaseSensitive)) {
+    if (windowName.isEmpty() || !windowName.compare(QStringLiteral("main"), Qt::CaseSensitive)) {
```

Comment:
⚠️ __I18n__ ⚠️  Although we are not internationalising the _Lua_ interface we may have to consider what term we use to describe the "main" window for non-English users as that may be a GUI issue as well.

:book: `QString::compare(...)` is analogous to the ___C___ library `strcmp` string comparison in that it returns an `int` that is `0` if the two __are equal__ (given the case sensitivity) and a value more or less than `0` otherwise - so the negation is needed to correctly detect the "main" console case...